build/Makefile

```diff
@@ -491,22 +508,36 @@ build-allocator-image: $(ensure-build-image) build-allocator-binary build-licens
 	docker build $(agones_path)/cmd/allocator/ --tag=$(allocator_tag) $(DOCKER_BUILD_ARGS)
 
 ifeq ($(WITH_WINDOWS), 1)
+ifeq ($(WITH_ARM64), 1)
 # push the gameservers sidecar image
-push-agones-sdk-image: push-agones-sdk-linux-image
+push-agones-sdk-image: push-agones-sdk-linux-image-arm64 push-agones-sdk-linux-image-amd64
+	# Ensure that the sidecar manifest is removed before creating a new one.
+	-DOCKER_CLI_EXPERIMENTAL=enabled docker manifest rm $(sidecar_tag)
+	# Using docker buildx with foreign OSes is a bit awkward. Since the layers but not the tag
+	# was cached we have to rebuild the image (which should be fast since it's cached).
+	$(MAKE) WINDOWS_DOCKER_PUSH_ARGS=--push build-agones-sdk-image-windows
+	# Agones should start using the -linux_amd64 suffix for Linux images. This allows multi-arch support in the future.
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(sidecar_tag) $(sidecar_linux_amd64_tag) $(sidecar_linux_arm64_tag) $(foreach windows_version, $(WINDOWS_VERSIONS), $(sidecar_tag)-windows_amd64-$(windows_version))
```

Comment:
Curious - do we still need `DOCKER_CLI_EXPERIMENTAL` here? or can it be removed like elsewhere?

Reply:
https://docs.docker.com/engine/reference/commandline/manifest/
It still marked as experimental

Reply:
https://docs.docker.com/engine/reference/commandline/manifest/
It still marked as experimental